src/templates/quote/quote.js

```diff
@@ -0,0 +1,90 @@
+import { React, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import './quote.scss';
+import BaseSlideExecution from '../baseSlideExecution';
+import Icon from './icon';
+
+/**
+ * Quote component.
+ *
+ * @param {object} props
+ *   Props.
+ * @param {object} props.slide
+ *   The slide.
+ * @param {object} props.content
+ *   The slide content.
+ * @param {boolean} props.run
+ *   Whether or not the slide should start running.
+ * @param {Function} props.slideDone
+ *   Function to invoke when the slide is done playing.
+ * @returns {JSX.Element}
+ *   The component.
+ */
+function Quote({ slide, content, run, slideDone }) {
+  const { quotes, quoteInTwoLines } = content;
+  const quoteClasses = quoteInTwoLines ? 'quote two-lines' : 'quote';
+  const [first] = quotes;
+  const [currentQuote, setCurrentQuote] = useState(first);
+  const [show, setShow] = useState(true);
+
+  /**
+   * Setup slide run function.
+   */
+  const slideExecution = new BaseSlideExecution(slide, slideDone);
+  useEffect(() => {
+    if (run) {
```

Comment:
I am sure that i'm missing something... but where is `run` defined?

Reply:
above 

`function Quote({ slide, content, run, slideDone }) {`